package.json

```diff
@@ -16,8 +16,10 @@
     "itest:node": "jest --selectProjects itest-node --coverage"
   },
   "dependencies": {
+    "@types/pako": "^2.0.3",
```

Comment:
This should only be used at build/test time. Can we move this to `devDependencies`?